error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -355,42 +355,6 @@ Optional<T> after(Optional<T> optional) {
     }
   }
 
-  /**
-   * Avoid unnecessary {@link Optional} to {@link Stream} conversion when filtering a value of the
-   * former type.
-   */
-  static final class OptionalFilter<T> {
```

Comment:
I suspect removing these two new Refaster rules in unintended? :slightly_smiling_face:

Reply:
Nice catch! Likely an error while moving the code to a new branch (I initially wrote this on top of #332).